modules/fedimint-ln-client/src/lib.rs

```diff
@@ -770,7 +773,7 @@ impl LightningClientModule {
             .min_final_cltv_expiry_delta(18)
             .payee_pub_key(node_public_key)
             .expiry_time(Duration::from_secs(
-                expiry_time.unwrap_or(DEFAULT_EXPIRY_TIME),
+                expiry_time.unwrap_or(DEFAULT_INVOICE_EXPIRY_TIME.as_secs()),
```

Comment:
**nitpick:** That conversion back and forth from/to `Duration` looks weird. Would `expire_time.map(Duration::from_secs).unwrap_or(DEFAULT_INVOICE_EXPIRY_TIME)` work? I would just do somewhere before:

```
let expire_time = expire_type.map(Duration::from_secs);
```

to get it of the way, then `expire_time.unwrap_or(DEFAULT_INVOICE_EXPIRY_TIME)` would work just perfectly.